www/front_src/src/Resources/Listing/index.tsx

```diff
@@ -107,7 +107,7 @@ const ResourceListing = (): JSX.Element => {
       loading={loading}
       columnConfiguration={columns}
       tableData={listing?.result}
-      currentPage={page - 1}
+      currentPage={(page || 1) - 1}
```

Comment:
why `-1` ? 
page => 1 - 1 = 0, so it's possible to have 0 value ?

Reply:
pages are 0 based in the Listing

Reply:
ok :+1: